test/models/user_test.rb

```diff
@@ -335,4 +335,194 @@ def test_notes_template_validation
     assert(u.invalid?,
            "Notes template with duplication headings should be invalid")
   end
+
+  def test_disable_account
+    rolf.disable_account
+    rolf.reload
+    assert_blank(rolf.password)
+    assert_blank(rolf.email)
+    assert_blank(rolf.mailing_address)
+    assert_blank(rolf.notes)
+    assert_true(rolf.blocked)
+  end
+
+  def test_delete_api_keys
+    assert_not_zero(ApiKey.where(user: rolf).count)
+    rolf.delete_api_keys
+    assert_zero(ApiKey.where(user: rolf).count)
+  end
+
+  def test_delete_interests
+    assert_not_zero(Interest.where(user: junk).count)
+    junk.delete_interests
+    assert_zero(Interest.where(user: junk).count)
+  end
+
+  def test_delete_notifications
+    assert_not_zero(Notification.where(user: rolf).count)
```

Comment:
I have no idea why `assert_not_zero` blows up here, and didn't want to take the time to figure it out. 
So I suggest
```suggestion
    assert_operator( 0, "<", Notification.where(user: rolf).count)
```

Reply:
I think you need `assert_not_nil` here. That seems to work for me (cloned the branch, trying it locally)

Reply:
1. While `assert_not_nil` avoids the Error, it can't be right.
The assertion (`assert_not_zero`) ensures that the fixtures are OK before proceeding with the rest of the test.
In particular, the test makes sense only if Rolf starts with some notifications.
`assert_not_nil` will give the wrong result (`true`) if Rolf has 0 Notifications.
2. `assert_not_zero` just blows up in this particular test method. 
With a breakpoint at line 362, we manually confirm that the `count` is correct:
```ruby
 Notification.where(user: rolf).count => 1
```
But `assert_not_zero` throws a NoMethodError:
```ruby
assert_not_zero(1) =>
ERROR UserTest#test_delete_notifications (230.16s)
Minitest::UnexpectedError:         NoMethodError: undefined method `assert_not_zero' for #<UserTest:0x0000555f48ca6578 @NAME="test_delete_notifications", @failures=[#<Minitest::UnexpectedError: Unexpected exception>], @assertions=1, @fixture_cache={"users"=>{"rolf"=>#<User 241228755: "Rolf Singer (rolf)">}}, @fixture_connections=[], @connection_subscriber=#<ActiveSupport::Notifications::Fanout::Subscribers::Timed:0x0000555f48ca4980 
...
Did you mean?  assert_not_same
               assert_not_nil
  (rdbg)//vagrant/mushroom-observer/test/models/user_test.rb:1:in `test_delete_notifications'

```
How did `assert_not_zero` get undefined just for this particular test? It works fine elsewhere in this file. See following screen shot.
<img width="444" alt="Screen Shot 2022-09-26 at 7 20 11 AM" src="https://user-images.githubusercontent.com/45460/192300906-805f30e5-e4a9-4030-8a66-babce1358c7f.png">

Reply:
I was wrong. `assert_zero` and `assert_not_zero` are not working in other places.
I think there's no such thing as `asert_zero` or `assert_not_zero`.
So these should probably be globally changed to
`assert_equal(0, ...` or `assert_operator( 0, "=", ...)` 
`assert_operator( 0, "<",  ...)`